docs/wiki/development/dependencies.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: wiki
+title: Dependencies
+description: This page details each component and its dependencies.
+group: development
+parent: wiki
+order: 0
+---
+
+This page details each component and the addons it requires.
+
+## 1. Removing a component
+
+Any component you wish to remove must also remove any component which depends on it.
+
+**Example:**
+
+To remove `ace_ai` you must also remove `ace_zeus`.
+
+Because `ace_zeus` is being removed you must also remove any components that require `ace_zeus` which in this case would be `ace_intelitems`.
+
+## 2. Dependencies
+
+{% assign pages_by_title = site.pages | sort: "title" %}
+{% for page in pages_by_title %}
+    {%- if page.group == 'feature' and page.component -%}
+        ### {{ page.title }}
+
+        {% capture component %}{{ page.component }}{% endcapture %}
+        {% include dependencies_list.md component=component %}
+
+        {%- if page.component == "common" or page.component == "main" or page.component == "interaction" or page.component == "interact_menu" -%}
```

Comment:
maybe add a frontmatter property to these pages, and check for it here

Reply:
It's very specific to those 4 and that won't change without a big rework, so I don't really see the point. It's easier to miss a frontmatter variable than this too.

Reply:
Another might be added.
I don't like hardcoding these things here.
That's all. So please change it 👍

Reply:
So is it a "maybe add" or "do add" now? Your wording confuses me between idea inquiry and statement.

Reply:
Yes, please don't check for hardcoded strings in code. Please use a property for it.